### ISSUE TITLE
[Frontend V2] Reduce padding below selected phase name and dropdown

### ISSUE THREAD

**Issue by Ram81** [Cloud-CV/EvalAI#3342]
### Description

Here's how the dropdown for the selected phase looks on the `Leaderboard` tab.

![Screen Shot 2021-03-24 at 12 06 21 PM](https://user-images.githubusercontent.com/16323427/112266128-9bff0b00-8c99-11eb-8bd1-a1177d6f6eca.png)

After a phase is selected the padding-bottom should be less for the selected option in the dropdown.

**Comment by Alabhya268** [comment]
@Ram81, Basically we need to reduce padding enough so dropdown icon and selected phase look even right ?

i want to work on this.

**Comment by Ram81** [comment]
@Alabhya268 you already picked #3336, it would be good if you work on 1 issue at a time and then move on to the next issue once the first one is done 🙂

**Comment by ankitpal1029** [comment]
@Ram81 what submission do i make so that it makes it to the leaderboard in order for me to replicate the padding issue.

**Comment by Alabhya268** [comment]
@Ram81 

> @Alabhya268 you already picked #3336, it would be good if you work on 1 issue at a time and then move on to the next issue once the first one is done 
> 
> > @Ram81, Basically we need to reduce padding enough so dropdown icon and selected phase look even right ?
> 
> yes, that's correct
@Ram81 since #3336 has already been merged can I work on this .

**Comment by Ram81** [comment]
>@Ram81 what submission do i make so that it makes it to the leaderboard in order for me to replicate the padding issue.

@ankitpal1029 you have to create a private phase to reproduce the issue.

>Can i work on this?

@ankitpal1029 as you are working on #3376 you can pick up another issue after you are done with it.


>@Ram81 since #3336 has already been merged can I work on this .

@Alabhya268 sure, go ahead

**Comment by Alabhya268** [comment]
![leaderboard](https://user-images.githubusercontent.com/57143358/113010872-d0416280-9196-11eb-826b-55214b4f9a66.png)

@Ram81, Padding looks fine in my system in the leaderboard tab. Could you please try to recreate this and explain under what circumstance this is happening.

**Comment by Alabhya268** [comment]
![Screenshot from 2021-03-30 20-39-22](https://user-images.githubusercontent.com/57143358/113012053-0af7ca80-9198-11eb-85b4-a9734cbe42f2.png)

@Ram81, It still looks fine even when the tag is private. Please look at the image above.

**Comment by Alabhya268** [comment]
@Ram81, did you checked it?

**Comment by RishabhJain2018** [comment]
Closing this since we are not working on Frontend_v2
